web/public/js/services/fallback-translate.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+angular.module('cpZenPlatform').factory('fallbackTranslate', ['$translate', function($translate) {
+  var originalTranslateInstant = $translate.instant;
+
+  $translate.instant = function (key) {
```

Comment:
Ah, this file slipped in there. It shouldn't be here at all. Doesn't even work. 😄